Assets/Scripts/Spawn System/Checkpoint.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Checkpoint : MonoBehaviour
+{
+    
+    public bool activated = false; //stops player from reactivation old check points
+    public Vector3 postion; //spawn location
```

Comment:
these should be private
You can create a private serialized field using the syntax [SerializeField] private bool activated